src/Particle.java

```diff
@@ -25,13 +26,16 @@ public Particle(double x, double y, int radius, double velocity, double angle, i
         this.ID = ID;
     }
 
-    /**Rysowanie czasteczki*/
+    /**
+     * Method that draws particle.
+     */
     @Override
     public void draw(Graphics g, Color color) {
         super.draw(g, color);
     }
 
-    /**Metoda odpowiedzialna za ruch czasteczki
+    /**
+     * Method that determines particle movement.
      * Wlaczane jest wygladzanie krawedzi, wybierany kolor wnetrza czasteczki, kolor ramki oraz wypisywane na niej wlasnego ID
      * Pilnowane rowniez jest to, aby czastka nie wyleciala za ramke

```

Comment:
#1 
@PawelLawrynowicz 
Comment seem improper, looks like they're remnants of another method. Should i translate them like all other or erase them?

Reply:
True that, these comments belong in [here](https://github.com/PawelLawrynowicz/ParticleCollider/blob/bbc7ecfa05d66ce71737e7c1fc8ce89de7213c25/src/RigidBody.java#L23)